### ISSUE TITLE
Some values from PermissionUtil return incorrect values

### ISSUE THREAD

**Issue by EricRabil** [DV8FromTheWorld/JDA#299]
I've noticed an odd bug, not really sure what's happening here...

```java
PermissionUtil.checkPermission(this.srv.getChatChannel(), this.srv.getSelfMember(), Permission.MANAGE_CHANNEL)
```

returns false; however,

```java
this.srv.getChatChannel().getManager().setTopic(chatTopic).complete();
```

goes through without an issue. Am I doing something wrong here, or is this a bug?

**Comment by DV8FromTheWorld** [comment]
It is quite odd that you're using permission util directly, just as a note.

That being said, yes, there is a bug with MANAGE_CHANNEL's lookup, @Almighty-Alpaca brought it up but it got lost in a sea of other development.

**Comment by EricRabil** [comment]
I don't understand, how else would I access it? There's no non-static
methods from what I'm looking at
Or do you mean usually JDA is the only thing that uses PermissionUtil

**Comment by MinnDevelopment** [comment]
There are `hasPermission` methods on all `IPermissionHolder` entities such as `Member` and `Role` which all use `PermissionUtil` in their implementation.

**Comment by EricRabil** [comment]
Ah okay, must've missed that going from JDA 2 to 3

On Tue, Mar 28, 2017 at 6:51 AM Florian Spieß <notifications@github.com>
wrote:

There are hasPermission methods on all IPermissionHolder entities such as
Member and Role which all use PermissionUtil in their implementation.

—
You are receiving this because you authored the thread.
Reply to this email directly, view it on GitHub
<https://github.com/DV8FromTheWorld/JDA/issues/299#issuecomment-289733260>,
or mute the thread

**Comment by MinnDevelopment** [comment]
This should be fixed as of 319aae5
Feel free to open again if any related issues appear.
